src/styles/utils/_vars.scss

```diff
@@ -1 +1,6 @@
 $c-gray: #eee;
+$colorRed: #d12d35;
+$colorBlack: #161616;
```

Comment:
Variable names should be consistent in naming convention. Consider changing '$colorBlack' to '$c-black' for consistency.